### PR TITLE
Few small changes

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
## About The Pull Request
<details>
<summary>
Adds .50 rubber speed loaders/mags to the sec-tech<br>
Adds a cautery to the SI advanced surgical kit from which it was missing<br>
readds the gear harness for synthetics and such.<br>
adds colorable tritail with sprites courtesy of Vesta-Bay!<br>
removes landmines from the sec-misc disk
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
FULLY tested! Breaks nothing! I PROMISE this

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm [hunk 1]
```diff
@@ -15,9 +15,10 @@
 		/datum/design/autolathe/tool/tacknife,
 		/datum/design/autolathe/tool/combat_shovel,
 		/datum/design/autolathe/sec/beartrap,
-		/datum/design/autolathe/device/landmine = 2, //can make 10 warcrimes
+		//datum/design/autolathe/device/landmine = 2, //can make 10 warcrimes //why did you put this here this was an awful idea.
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:18]:
```suggestion
		/datum/design/autolathe/device/landmine = 2, //can make 10 warcrimes
```

Comment (cdb-is-not-good) [reply code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:18]: please god get rid of the landmines on the misc disk it makes no sense to make explosives so easily accessible by everyone

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm [hunk 2]
```diff
@@ -15,9 +15,10 @@
...
 		/datum/design/autolathe/sec/silencer,
 		/datum/design/autolathe/sec/gun_case,
+		/obj/item/storage/secure/briefcase,
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:21]: This brakes the sec disk

File: code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/autolathe_disk/guns_ammo_sec_disks.dm:18]: Dont remember landmines

**Pull request merged**